src/deprecates.jl

```diff
@@ -23,9 +23,9 @@ import Base.varm, Base.stdm
 if !isdefined(Base, :stderr)
     @deprecate stderr(obj::StatisticalModel) stderror(obj)
 else
-    function (io::IO)(obj::StatisticalModel)
+    function (io::typeof(stderr))(obj::StatisticalModel)
```

Comment:
The problem with this is that `typeof(stderr)` can change at runtime if `redirect_stderr` is called.

So this should just be `::IO` to catch all usages.   I don't think it is a problem to have a fairly general method here as a temporary deprecation since it continues to throw a `MethodError` for non-`stderr` objects. (People are unlikely to pass `StatisticalModel` objects to random `IO` objects by accident anyway.)